src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2808,6 +2815,35 @@ private void PopulateStandaloneSignatures()
             }
         }
 
+        private int[] SerializeThrowNullMethodBodies(BlobBuilder ilBuilder, out Blob mvidStringFixup)
```

Comment:
Could be done with a regular ```if``` statement though. i don't remember seen any other place in the code base where we add a parameter just for this purpose.